docs/upgrading-to-integration.asciidoc

```diff
@@ -0,0 +1,167 @@
+
+[discrete]
+[[apm-integration-upgrade-steps]]
+=== Upgrade a self-managed cluster
+
+. <<apm-integration-upgrade-1>>
+. <<apm-integration-upgrade-2>>
+. <<apm-integration-upgrade-3>>
+. <<apm-integration-upgrade-4>>
+. <<apm-integration-upgrade-5>>
+. <<apm-integration-upgrade-6>>
+
+[discrete]
+[[apm-integration-upgrade-1]]
+==== Upgrade the Elastic Stack
+
+The Elastic Stack ({es} and {kib}) must be upgraded to version 7.0 or higher.
+See the {stack-ref}/upgrading-elastic-stack.html[Elastic Stack Installation and Upgrade Guide] for guidance.
+
+Review the APM <<release-notes,release notes>>, <<apm-breaking,breaking changes>>,
+and Observability {observability-guide}/whats-new.html[What's new] content for important changes between
+your current APM version and this one.
+
+[discrete]
+[[apm-integration-upgrade-2]]
+==== Add a {fleet} Server
+
+{fleet} Server is a component of the {stack} used to centrally manage {agent}s.
+The APM integration requires a {fleet} Server to be running and accessible to your hosts.
+Add a {fleet} Server by following {fleet-guide}/add-a-fleet-server.html[this guide].
+
+TIP: If you're upgrading a self-managed deployment of the {stack}, you'll need to enable
+{ref}/configuring-stack-security.html[{es} security] and the
+{ref}/security-settings.html[API key service].
+
+After adding your {fleet} Server host and generating a service token, the in-product help in {kib}
+will provide a command to run to start an {agent} as a {fleet} Server.
+Commands may require administrator privileges.
+
+Verify {fleet} Server is running by navigating to **Fleet** > **Agents** in {kib}.
+
+[discrete]
+[[apm-integration-upgrade-3]]
+==== Install a {fleet}-managed {agent}
+
+NOTE: It's possible to install the Elastic APM integration on the same {agent} that is running the {fleet} Server integration. For this use case, skip this step.
+
+The {fleet}-managed {agent} will run the Elastic APM integration on your edge nodes, next to your applications.
+To install a {fleet}-managed {agent}, follow {fleet-guide}/install-fleet-managed-elastic-agent.html[this guide].
+
+[discrete]
+[[apm-integration-upgrade-4]]
+==== Add the APM integration
+
+The APM integration receives performance data from your APM agents,
+validates and processes it, and then transforms the data into {es} documents.
+
+To add the APM integration, see <<add-apm-integration>>.
+Only complete the linked step (not the entire quick start guide).
+If you're adding the APM integration to a {fleet}-managed {agent}, you can use the default policy.
+If you're adding the APM integration to the {fleet} Server, use the policy that the {fleet} Server is running on.
+
+TIP: You'll configure the APM integration in this step.
+See <<input-apm>> for a reference of all available settings.
+Make note of the URL where APM Server is listening, and if applicable, the secret token;
+you'll need these in the next step.
+
+[discrete]
+[[apm-integration-upgrade-5]]
+==== Update APM agents
```

Comment:
This should not be necessary. As long as they configure the apm integration with the same secret token or api keys enabled and the same host - the agents should not require any changes.

Reply:
_**d0h**_. Makes sense. Updated.